### ISSUE TITLE
taskbar icon missing in Ubuntu Budgie 18.04

### ISSUE THREAD

**Issue by stragu** [nextcloud/desktop#905]
### Expected behaviour
There is a personalised Nextcloud icon for the taskbar.

### Actual behaviour
I can see the default icon that appears when an icon is missing (second from the left):
![image](https://user-images.githubusercontent.com/1747497/49217172-9de19800-f418-11e8-851d-5f0701e05aec.png)
The OwnCloud client has an icon (first from the left).

### Steps to reproduce
1. Install Nextcloud client
2. Open Nextcloud client and close to taskbar

### Client configuration
Client version: 2.3.3

Operating system: Tuxedo Budgie (equivalent to Ubuntu Budgie) 18.04

OS language: English (United Kingdom)

Qt version used by client package (Linux only, see also Settings dialog): not specified in "General" tab.

Client package (From Nextcloud or distro) (Linux only): from Nextcloud, provided by http://download.opensuse.org/repositories/isv:/ownCloud:/desktop/Ubuntu_18.04

Installation path of client: 
```
whereis owncloud-client
owncloud-client: /etc/owncloud-client
```

### Logs

I assume the log won't be useful here but happy to provide it if you think otherwise.

**Comment by juliushaertl** [comment]
Can you please check if this is still the case with the new client version 2.5.0? It should work fine there.

**Comment by stragu** [comment]
Hi @juliushaertl 

Really sorry about that terrible bug report!!! I got the source wrong: it actually comes from the Software app, as it is the most recent snap offered. And that version (2.3.3) says that there is no newer version available, which made me not look further. (it is called "nextcloud-client" in the Software app, version 2.3.3+gitab40efe, but if I do an `apt-cache policy` on it, it doesn't find it – is that because it's a snap?)

I just added https://launchpad.net/~nextcloud-devs/+archive/ubuntu/client to my sources, installed 2.5.1 and yes: the taskbar icon is fine now :)

Thank you for your reply! :+1:
